.vib/apache/vib-verify.json

```diff
@@ -1,51 +1,53 @@
 {
   "phases": {
     "package": {
-      "context": {
-        "resources": {
-          "url": "{SHA_ARCHIVE}",
-          "path": "/bitnami/apache"
-        }
-      },
       "actions": [
         {
           "action_id": "helm-package"
         },
         {
           "action_id": "helm-lint"
         }
-      ]
-    },
-    "verify": {
+      ],
       "context": {
```

Comment:
In the previous version, there was a `verify` before the `context`. 
Now, I don't see it. Could it affect the functionality of the pipeline?

Reply:
It shouldn't be removed but ordered, from
```
{
    "phases": {
      "package": {
        "context": ...,
        "actions": ...
      },
      "verify": {
        "context": ...,
        "actions": ...
      }
    }
  }
```
to
```
{
  "phases": {
    "package": {
      "actions": ...,
      "context": ...
    },
    "verify": {
      "actions": ...,
      "context": ...
    }
  }
}
```